core/src/main/java/tc/oc/pgm/observers/ObserverToolsMatchModule.java

```diff
@@ -36,13 +34,6 @@
 @ListenerScope(MatchScope.LOADED)
 public class ObserverToolsMatchModule implements MatchModule, Listener {
 
-  public static class Factory implements MatchModuleFactory<ObserverToolsMatchModule> {
-    @Override
-    public ObserverToolsMatchModule createMatchModule(Match match) throws ModuleLoadException {
-      return new ObserverToolsMatchModule(match);
-    }
-  }
-
   // Slot where tool item is placed
```

Comment:
This doesn't seem like it belongs in this commit

Reply:
Oh whoops 🤦‍♂